examples/torch/semantic_segmentation/main.py

```diff
@@ -207,12 +206,9 @@ def create_train_data_loader(batch_size_):
             shuffle=train_shuffle)
     # Loaders
     train_loader = create_train_data_loader(batch_size)
+    init_loader = train_loader
     if config.batch_size_init:
         init_loader = create_train_data_loader(config.batch_size_init)
-    else:
-        init_loader = deepcopy(train_loader)
-    if config.distributed:
-        init_loader.num_workers = 0  # PyTorch multiprocessing dataloader issue WA
```

Comment:
AFAIK, there's the same fix for init data loaders in tests. Why not to revert it there as well? 
That might be a prove that fix can be safely reverted.

Reply:
Good idea, reverted this.